src/features/usage-consolidation-accordion/usage-consolidation-content-package.js

```diff
@@ -3,46 +3,76 @@ import PropTypes from 'prop-types';
 
 import NoCostPerUseAvailable from './no-cost-per-use-available';
 import SummaryTable from './summary-table';
+import TitlesTable from './titles-table';
 import {
   costPerUse as costPerUseShape,
   entityTypes,
+  costPerUseTypes,
 } from '../../constants';
 
 const propTypes = {
   costPerUseData: costPerUseShape.CostPerUseReduxStateShape.isRequired,
+  onLoadMoreTitles: PropTypes.func.isRequired,
+  onViewTitles: PropTypes.func.isRequired,
   year: PropTypes.number.isRequired,
 };
 
-const UsageConsolidationContentPackage = ({
-  costPerUseData,
-  year,
-}) => {
+const UsageConsolidationContentPackage = props => {
+  const data = props.costPerUseData.data[costPerUseTypes.PACKAGE_COST_PER_USE];
+  if (!data) {
+    return null;
+  }
+
   const {
     cost,
     costPerUse,
     usage,
-  } = costPerUseData.data?.attributes?.analysis;
+  } = data?.attributes?.analysis;
 
   const noCostPerUseAvailable = !cost && !costPerUse && !usage;
 
-  const customProperties = {
-    columnMapping: { cost: 'ui-eholdings.usageConsolidation.summary.packageCost' },
+  const handleFetchNextPage = (page, pageSize, sortedColumn, sortOrder) => {
+    props.onLoadMoreTitles({
+      page,
+      pageSize,
+      sort: sortedColumn,
+      order: sortOrder,
+    });
   };
 
-  return noCostPerUseAvailable
-    ? (
-      <NoCostPerUseAvailable
-        entityType={entityTypes.PACKAGE}
-        year={year}
-      />
-    )
-    : (
+  const handleSortTitles = (sortedColumn, sortOrder) => {
+    props.onViewTitles({
+      sort: sortedColumn,
+      order: sortOrder,
+    });
+  };
+
+  return noCostPerUseAvailable ? (
+    <NoCostPerUseAvailable
+      entityType={entityTypes.PACKAGE}
+      year={props.year}
+    />
+  ) : (
+    <>
       <SummaryTable
         id="packageUsageConsolidationSummary"
-        costPerUseData={costPerUseData}
-        customProperties={customProperties}
+        entityType={entityTypes.PACKAGE}
+        customProperties={{
+          columnMapping: { cost: 'ui-eholdings.usageConsolidation.summary.packageCost' },
+        }}
+        noCostPerUseAvailable={noCostPerUseAvailable}
+        costPerUseType={costPerUseTypes.PACKAGE_COST_PER_USE}
+        onViewTitles={props.onViewTitles}
+        costPerUseData={props.costPerUseData}
+        {...props}
```

Comment:
If you destructuring props you could omit declaration of previous two
```suggestion
        {...props}
```